pkg/ingester/ingester.go

```diff
@@ -3764,6 +3758,39 @@ func (i *Ingester) ShutdownHandler(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// startReadRequest tries to start a read request.
+// If it was successful, startReadRequest returns a function that should be
+// called to finish the started read request once the request is completed.
+// If it wasn't successful, the causing error is returned. In this case no
+// function is returned.
+func (i *Ingester) startReadRequest() (func(error), error) {
+	var (
+		start                        = time.Now()
+		err                          error
+		acquiredCircuitBreakerPermit bool
+	)
+	acquiredCircuitBreakerPermit, err = i.circuitBreaker.tryAcquirePermit()
```

Comment:
I think the var declaration isn't needed here, we can just do this:
```suggestion
	start := time.Now()
	acquiredCircuitBreakerPermit, err := i.circuitBreaker.tryAcquirePermit()
```

Reply:
I have applied the proposed changes.